_pages/algorithm-development/ci-cd.md

```diff
@@ -9,6 +9,11 @@ author: jpeck
 image:
   teaser: /language_logos/jenkins.png
 permalink: /algorithm-development/ci-cd/
+menus:
+  algo_development:
+    url: /developers/algorithm-development/ci-cd
+    title: "Deploy Models via Jenkins CI/CD or GitHub Actions"
```

Comment:
<img width="354" alt="Screen Shot 2019-11-19 at 3 48 41 PM" src="https://user-images.githubusercontent.com/8980868/69196711-1804a400-0ae4-11ea-854f-fd41b71ad6af.png">
In order to get this whole thing to be visible I need to bump the width of `.syn-page-header-nav` from 15rem to 25rem. Wondering if it'd be simpler to just rename this one to 'CI/CD'

Reply:
Huh, that should be wrapping. Do you have synapse updated and built?

Reply:
Yep, I've been nuking the `dist/` folder and rebuilding for each of these PRs. I'll take a look first thing tomorrow to verify.

Reply:
Curious!

Reply:
@bnlb are you still seeing this?

Reply:
Nope. I see it wrapping now